docs/libraries/libraries-with-cli.md

```diff
@@ -0,0 +1,660 @@
+# Writing Libraries with Cross Platform Tools
+
+By [Phillip Carter](https://github.com/cartermp)
+
+**Some details are subject to change as the toolchain evolves.**
+
+Targeting .NET Core for libraries can be done a number of ways, depending on the platform(s) you wish to support.  Sometimes Visual Studio is not available or a preferred development option.  .NET Core has a set of command line tools you can use instead of Visual Studio.
```

Comment:
Not sure how I feel about the last part of the last sentence.  I don't think that people will be concerned if I call out that that they are the tools VS will use directly.